test/cli/simple-globals.json

```diff
@@ -0,0 +1,11 @@
+[{
```

Comment:
Change it to standard. In fact, utilise a single data source for environment and globals - implying revert addition of new file.